src/dlgTriggerEditor.cpp

```diff
@@ -125,68 +119,56 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     setStatusBar( statusBar );
     statusBar->show();
     mIsGrabKey = false;
-    QVBoxLayout * pVB1 = new QVBoxLayout(mainArea);
+
+    // Replacement code to use single splitter for all right widgets:
+    mpSystemMessageArea = new dlgSystemMessageArea(splitter_rightTopBottom);
+    mpTriggersMainArea = new dlgTriggersMainArea(splitter_rightTopBottom);
+    mpTimersMainArea = new dlgTimersMainArea(splitter_rightTopBottom);
+    mpAliasMainArea = new dlgAliasMainArea(splitter_rightTopBottom);
+    mpActionsMainArea = new dlgActionMainArea(splitter_rightTopBottom);
+    mpKeysMainArea = new dlgKeysMainArea(splitter_rightTopBottom);
```

Comment:
It should not be a fixed size, but rather a Qt sizing property - remember those prefer/minimum/expanding. That will make it work with HiDPI displays, and other OS's. Remember not every OS uses the same sizes for widgets!